test/upgrade/operator/201_customer_tablets.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: planetscale.com/v2
+kind: VitessCluster
+metadata:
+  name: example
+spec:
+  images:
+    vtctld: vitess/lite:latest
+    vtgate: vitess/lite:latest
+    vttablet: vitess/lite:latest
+    vtbackup: vitess/lite:latest
+    mysqld:
+      mysql56Compatible: vitess/lite:latest
```

Comment:
Shouldn't these be set to `v13.0.0` like `cluster_upgrade.yaml`?

Reply:
Correct. good catch

Reply:
fixed